docker-compose.yml

```diff
@@ -44,9 +34,7 @@ services:
     ports:
       - ${EXPOSED_PGPORT:-15432}:${PGPORT:-5432}
```

Comment:
Using only default values, I initially got a mismatch for the exposed DB port (15432 / 5432). I suggest to change it here (in the docker-compose) to 5432 since the default value is 5432 everywhere else

Reply:
I'm not sure I got what you mean by that.

Reply:
I meant replace all occurrences of `15432` to `5432` (I couldn't do it with code suggestion as you did not edited this part in your PR). The goal would be that the build works with the specified default values (without needing to specify a port)
I can do it in another PR if necessary

Reply:
Ahh-got it. I will change that :)

Reply:
I did update. Please have a look :)